ping/ping_common.c

```diff
@@ -401,6 +401,10 @@ int pinger(struct ping_rts *rts, ping_func_set_st *fset, socket_st *sock)
 		/* Socket buffer is full. */
 		tokens += rts->interval;
 		return MIN_INTERVAL_MS;
+	} else if (errno == EMSGSIZE) {
+		/* SOCK_DGRAM fails on -s > 65527 */
```

Comment:
I think the comment is confusing since SOCK_RAW also fails. It will also be confusing after '-s' gets clamped down.
How about changing it to: "for example, sendto with len > 65527 on SOCK_DGRAM fails with this errno"

Reply:
Thanks fixed!